### PR TITLE
Cluster Serving flink serving, correctness

### PR DESCRIPTION
**Pull request by Litchilitchy**

### CODE REVIEW

File: scripts/cluster-serving/cluster-serving-start [hunk 1]
```diff
@@ -81,10 +81,10 @@ else
...
 
 # try to start tensorboard
-if [[ ! -e "TensorboardEventLogs" ]]; then
-    mkdir TensorboardEventLogs
-fi
-tensorboard --logdir ./TensorboardEventLogs --port 6006 --host 0.0.0.0 &
+#if [[ ! -e "TensorboardEventLogs" ]]; then
+#    mkdir TensorboardEventLogs
+#fi
+#tensorboard --logdir ./TensorboardEventLogs --port 6006 --host 0.0.0.0 &
```
Comment (jason-dai) [inline scripts/cluster-serving/cluster-serving-start:87]: why disable Tensorboard?

File: scripts/cluster-serving/cluster-serving-start [hunk 2]
```diff
@@ -124,4 +128,4 @@ if [ "${params_performance_mode}" == "ON" ] && [ "${spark_master}" == "local[*]"
     fi
 fi
 
-${SPARK_HOME}/bin/spark-submit --master ${spark_master} --driver-memory ${spark_driver_memory} --executor-memory ${spark_executor_memory} --num-executors ${spark_num_executors} --executor-cores ${spark_executor_cores} --total-executor-cores ${spark_total_executor_cores} --conf "spark.executor.extraJavaOptions=-Dbigdl.engineType=${params_engine_type}" --conf "spark.driver.extraJavaOptions=-Dbigdl.engineType=${params_engine_type}" --jars ${SPARK_REDIS_JAR},${BIGDL_JAR} --class com.intel.analytics.zoo.serving.ClusterServing ${ZOO_JAR} >> cluster-serving.log
+${FLINK_HOME}/bin/flink run -c com.intel.analytics.zoo.serving.ClusterServing -p 1 ${ZOO_JAR}
```
Comment (jason-dai) [inline scripts/cluster-serving/cluster-serving-start:131]: how does the user specify cluster address? does it support YARN and K8s?

**Comment by jason-dai** [comment]
Can we make sure all the existing scripts and user document (e.g., https://analytics-zoo.github.io/master/#ClusterServingGuide/ProgrammingGuide/, https://analytics-zoo.github.io/master/#ProgrammingGuide/k8s/#launch-analytics-zoo-cluster-serving) work exactly as before?

**Comment by Litchilitchy** [comment]
> Can we make sure all the existing scripts and user document (e.g., https://analytics-zoo.github.io/master/#ClusterServingGuide/ProgrammingGuide/, https://analytics-zoo.github.io/master/#ProgrammingGuide/k8s/#launch-analytics-zoo-cluster-serving) work exactly as before?

tested on k8s, will test on yarn soon.

File: scripts/cluster-serving/cluster-serving-start (hunk 1 above)
Comment (Litchilitchy) [reply scripts/cluster-serving/cluster-serving-start:87]: flink log is available at flink ui, including throughput and record number, and by the way using driver to write log in flink is not quite feasible (up to my research so far)

Comment (jason-dai) [reply scripts/cluster-serving/cluster-serving-start:87]: Have we updated the document for these changes?

Comment (Litchilitchy) [reply scripts/cluster-serving/cluster-serving-start:87]: Updating in another PR

**Pull request merged**